dts/bindings/led_strip/worldsemi,ws2812-rpi_pico-pio.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2023, TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Worldsemi WS2812 LED strip, RaspberryPi Pico's PIO binding
```

Comment:
I am renewing my question: is the PIO mechanism only available on the RaspberryPi Pico device ? If not then maybe we can remove the "pico" string from the driver name and from other places in DT ?

Reply:
> is the PIO mechanism only available on the RaspberryPi Pico device 

Yes it is. PIO is only available on Rpi-Pico.
But "PIO" is easy to conflict name. I think it is better to clarify the context.